llama_stack/providers/impls/meta_reference/inference/generation.py

```diff
@@ -159,6 +158,16 @@ def build(
                 model = Transformer(model_args)
                 model = convert_to_int4_quantized_model(model, model_args, config)
                 model.load_state_dict(state_dict, strict=True)
+
+                if config.quantization.spinquant:
```

Comment:
we need to change this and also the "scheme" thing now because it has to come from `model_args`